src/deepsparse/server/build_logger.py

```diff
@@ -190,7 +237,32 @@ def _build_custom_logger(logger_arguments: Dict[str, Any]) -> BaseLogger:
     return logger
 
 
-def _get_target_identifier(endpoint_name: str, target_name: str) -> str:
+def _get_target_identifier(
+    *identifier_strings, is_system_logging_identifier=False
+) -> str:
+    if is_system_logging_identifier:
+        if len(identifier_strings) != 1:
+            raise ValueError(
+                "System logging target identifier must be a single string. "
+                f"Got {identifier_strings} instead."
+            )
+        return _get_target_identifier_system(*identifier_strings)
+    else:
+        if len(identifier_strings) != 2:
+            raise ValueError(
+                "Target identifier must be a tuple of two strings. "
+                f"Got {identifier_strings} instead."
+            )
+        return _get_target_identifier_data(*identifier_strings)
+
+
+def _get_target_identifier_system(group_name: str) -> str:
+    # TODO: Let's discuss how the support for regex would
```

Comment:
@bfineran let's discuss in more detail this feature.

Reply:
why do we need to prepend with `category`? instead of something like `*./group_name/.*`?

Reply:
I think this was initially introduced by you here: https://github.com/neuralmagic/deepsparse/pull/697/commits/780b38aa685a33d934a82a36dc641d8f2ab91db1

My suggestion is, let's land this PR ignoring the identifier convention, this will be something I will clean up in the scope of 1.4. in a separate PR